### DISCUSSION TITLE
When sharing an account, the logo does not show

### DISCUSSION THREAD

**Discussion by phendev** [community/community#23070] [New to GitHub]
<p>Hi all. When I send someone a link to the company profile, it shows the default picture - avatar.</p>
<p>I have a logo in my personal profile and company profile. Custom avatar.</p>
<p><img src="https://user-images.githubusercontent.com/85644033/181074677-cfe571f6-c192-403c-b138-d29f7c295674.jpeg" alt="1657874635835" data-base62-sha1="h7kskzuIAO3K3aJ5vaKoovORBzg" width="632" height="428"></p>
<p>Do I have to wait for the robot to index the new picture?</p>

**Comment by jsoref** [comment, marked as answer]
<p>Roughly yes.<br>
It looks fine here now:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/Phenomenon-Studio">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/Phenomenon-Studio" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/2119212/181074678-4b24e8b3-9d73-45d8-90d4-37ea130bffbf.png" class="thumbnail onebox-avatar" width="160" height="160">

<h3><a href="https://github.com/Phenomenon-Studio" target="_blank" rel="noopener">Phenomenon studio</a></h3>

  <p>Full-service studio for startups and established businesses - Phenomenon studio</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/6/f63e3e212579481f4963db9c6cc9722149041449.jpeg" data-download-href="/uploads/short-url/z8mOlEMnoh5yOEMQZyvADGSZ1dD.jpeg?dl=1" title="image"><img src="https://user-images.githubusercontent.com/2119212/181074679-5ffa643d-bfc1-4e17-b68c-8cf7253a3f07.jpeg" alt="image" data-base62-sha1="z8mOlEMnoh5yOEMQZyvADGSZ1dD" width="690" height="286" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/6/f63e3e212579481f4963db9c6cc9722149041449_2_690x286.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/f/6/f63e3e212579481f4963db9c6cc9722149041449_2_1035x429.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/f/6/f63e3e212579481f4963db9c6cc9722149041449_2_1380x572.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/6/f63e3e212579481f4963db9c6cc9722149041449_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1456×604 121 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>just tested it on my end too, as you can see, it’s your custom avatar too, maybe when you tested that last time it was cached or not yet updated</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/Phenomenon-Studio/.github">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/Phenomenon-Studio/.github" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181074683-f7cde42d-9cd0-4c81-80af-e5e9c69559ce.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/e/1eccf4898a275405a645455ac4e553634025046a_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/e/1eccf4898a275405a645455ac4e553634025046a_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/e/1eccf4898a275405a645455ac4e553634025046a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/e/1eccf4898a275405a645455ac4e553634025046a_2_10x10.png"></div>

<h3><a href="https://github.com/Phenomenon-Studio/.github" target="_blank" rel="noopener">GitHub - Phenomenon-Studio/.github</a></h3>

  <p>Contribute to Phenomenon-Studio/.github development by creating an account on GitHub.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
